activities/Day4_activities_answer_key.R

```diff
@@ -14,18 +14,14 @@ ggplot(animals_tb) +
 
 #3. We decide that our plot would look better with the animal names ordered from slowest to fastest. Using the animals_tb data frame, reorder the animals on the x-axis to start with the slowest animal on the left-hand side of the plot to the fastest animal on the right-hand side of the plot by completing the following steps:
 
-#a. Use the `arrange()` function to order the rows by speed from slowest to fastest and save to `animals_arranged`.
+#a. Use the `arrange()` function to order the rows by speed from slowest to fastest. Save the new variable as `animals_arranged`.
 animals_arranged <- animals_tb %>% arrange(speed)
```

Comment:
Here, would it be easier if they avoid creating the new df/tibble called anaimals_arranged and simply just create the names_ordered_by_speed  vector?

`names_ordered_by_speed <- animals_tb %>% 
arrange(speed) %>% 
select(animal_names`